patches/net/minecraft/client/multiplayer/ClientPacketListener.java.patch

```diff
@@ -54,6 +70,18 @@
  
              if (p_337415_ instanceof CommandBlockEntity && this.minecraft.screen instanceof CommandBlockEditScreen) {
                  ((CommandBlockEditScreen)this.minecraft.screen).updateGui();
+@@ -1420,9 +_,9 @@
+         }
+     }
+ 
+-    private void startWaitingForNewLevel(LocalPlayer p_304688_, ClientLevel p_304528_, ReceivingLevelScreen.Reason p_341690_) {
++    private void startWaitingForNewLevel(LocalPlayer p_304688_, ClientLevel p_304528_, ReceivingLevelScreen.Reason p_341690_, @Nullable ResourceKey<Level> toDimension, @Nullable ResourceKey<Level> fromDimension) {
```

Comment:
Vanilla methods should never be deleted, even if they are private. Therefore the vanilla method should remain as deprecated with a comment (i.e. `/** @deprecated Neo: use {@link #startWaitingForNewLevel(<insert param types here>)} instead */`) and forward to the Neo-added one with default values for the added params (i.e. null in this case)